torchci/lib/bot/mergeBot.ts

```diff
@@ -2,24 +2,41 @@ import { Probot } from "probot";
 import { addComment, reactOnComment } from "./botUtils";
 
 function mergeBot(app: Probot): void {
-  const mergeCmdPat = new RegExp("^\\s*@pytorch(merge|)bot\\s+(force\\s+)?merge\\s+this");
+  const mergeCmdPat = new RegExp(
+    "^\\s*@pytorch(merge|)bot\\s+(force\\s+)?merge\\s+this"
+  );
   const revertCmdPat = new RegExp("^\\s*@pytorch(merge|)bot\\s+revert\\s+this");
-  const rebaseCmdPat = new RegExp("^\\s*@pytorch(merge|)bot\\s+rebase\\s+(me|this)");
+  const rebaseCmdPat = new RegExp(
+    "^\\s*@pytorch(merge|)bot\\s+rebase\\s+(me|this)"
+  );
+  const mergeOnGreenCmdPat = new RegExp(
+    "^\\s*@pytorch(merge|)bot\\s+mergeOnGreen\\s+(me|this)"
```

Comment:
I mean this can be anything but could it be 
```suggestion
    "^\\s*@pytorch(merge|)bot\\s+merge\\s+(me|this)\\s*on\\s+green"
```

so someone could type "@pytorchbot merge this on green"? 

Also, it might be even better to combine this into the mergeCmdPat